controller/scripts/csi_pb2.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+set -x
+
+version="$1" # currently k8s on v1.2.0
```

Comment:
then let's keep the version value hardcoded for now, because it is important info, and comments are more likely to be removed

Reply:
done